.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
 Please contribute feedback on the overall design of WebAssembly, issues about
 the textual specification, or typos from the docs section of webassembly.org.
 
-For browser issues or tooling bugs, see links at http://webassembly.org/community/feedback/.
\ No newline at end of file
+For browser issues or tooling bugs, see links at https://github.com/WebAssembly/website/blob/master/community/feedback.md.
```

Comment:
> @jfbastien I do not believe the CG has 'consensus that was previously reached', there are many important matters of business for the CG in the roadmap and I did not see a heads up on these and thus I believe linking to these would misrepresent the CG.

The history is [here](https://github.com/WebAssembly/website/commits/master/roadmap.md). As noted on that page, there were plenty of announcements after that pull request had been outstanding for close to a week:

> October 2016 - Browser Preview announced with multiple interoperable implementations [[1](https://blogs.windows.com/msedgedev/2016/10/31/webassembly-browser-preview/)] [[2](http://v8project.blogspot.com/2016/10/webassembly-browser-preview.html)] [[3](https://hacks.mozilla.org/2016/10/webassembly-browser-preview)]

These announcement were ample dissemination, and their main goal was to [ask for feedback](http://webassembly.org/community/feedback/). Please, provide feedback! We've gotten plenty from others.

However, this pull request isn't the right place for feedback: this PR links to the roadmap. Changing the roadmap isn't a valid reason to block a link from being added. The link is valid now, it'll be valid after any roadmap changes. Adding this one link here won't suddenly make that page visible to the world: it's been visible for almost two months.

This is the first time, almost two months after the announcements, that I hear concerns about the roadmap from *anyone*. There's plenty to be concerned about w.r.t. its optimism about things being "ready", but it seems folks think the roadmap is pretty sensible and can slip a bit without much bother. Now's still the time to voice your opinion, but again, please not on unrelated PRs.

> Sorry I do not find your views on this matter believable and I note you have failed to respond to repeated requests to explain your claim to a position of Chair or to return this Chair to the CG and that moving to a W3C working group would take this decision away from the CG members.

Because this is all unrelated. I don't claim to be a chair.

> Obviously this is all heading to the CG mailing list for the CG members to sort out, but the members may not be kind if good faith attempts to do the right thing have not been made, and I am happy to give people more time.

The CG's chosen method of communication is github. Please use the CG's chosen method of communication.